packages/@aws-cdk/aws-rds/README.md

```diff
@@ -240,7 +236,6 @@ const role = new iam.Role(stack, 'RDSDirectoryServicesRole', {
 });
 const instance = new rds.DatabaseInstance(stack, 'Instance', {
   engine: rds.DatabaseInstanceEngine.mysql({ version: rds.MysqlEngineVersion.VER_8_0_19 }),
-  masterUsername: 'admin',
   vpc,
   domain: 'd-????????', // The ID of the domain for the instance to join.
   domainRole: role, // Optional - will be create automatically if not provided.
```

Comment:
i think it would be helpful to see a different usage of `Login` than just `fromUsername()` in the ReadMe.

Reply:
I agree, but I struggled to find the right place in the README to insert it. I'll give it another pass.